frontend/src/hooks/api/workspaceDocument.ts

```diff
@@ -70,7 +70,10 @@ export const useGetDocumentQuery = (workspaceId?: string | null, documentId?: st
 		};
 	}, [dispatch, query.data]);
 
-	return query;
+	const isError =
+		query.error && axios.isAxiosError(query.error) && query.error.response?.status === 500;
```

Comment:
It just handle 500 errors. So, the name `isError` contains more wide meaning.

Reply:
I Fixed it!